src/notification/slack-notify.ts

```diff
@@ -6,55 +6,44 @@ import { IFetch } from '../shared/fetch';
 import { fetchInjectSymbol } from '../shared/types';
 import { NotificationComment, SlackAttachment, SlackMessage } from './types';
 
-export function getMessage(
+function getMessage(
   deployment: MinardDeployment,
   previewUrl: string,
-  projectUrl: string,
-  branchUrl: string,
+  _projectUrl: string, // leaving in to have a similar structure to Flowdock notifications
```

Comment:
Since `getMessage` is only used inside this file, there's no need to keep the format of the function the same as in some other file. I'd just remove these two parameters.